packages/reactivity/src/computed.ts

```diff
@@ -20,6 +20,50 @@ export interface WritableComputedOptions<T> {
   set: ComputedSetter<T>
 }
 
+class _ComputedRef<T> {
```

Comment:
Instead of prefixing these internal classes with `_`, I prefer more explicit names like `ComputedRefImpl` or `ComputedRefConstructor`